src/ui/components/shared/LoadingScreen.tsx

```diff
@@ -28,49 +29,67 @@ export function LoadingScreenTemplate({ children }: { children?: ReactNode }) {
   }, [hoverboardColor]);
```

Comment:
This should be triggering a Lint failure. This effect is missing a dependency (`changeHoverboardColor`)

You could fix this in a variety of ways:
1. Memoize `changeHoverboardColor` with `useCallback`
2. Or just add the inline callback dependency (and maybe change `useEffect` to use `setTimeout` so it's clearer what it's meant to be doing– one delayed change per render, at most)